add diagram for new transreg flow

Willi-Pindjukov opened the pull request:
added the diagram for the new transreg flow to use it in the documentation

Willi-Pindjukov committed:
add diagram for new transreg flow

added the diagram for the new transreg flow to use it in the documentation

MaxBreida reviewed (changes requested):
1. Das Styling ist nen anderes als unsere anderen Diagramme... 😄 
2. Fände es besser wenn hier das Naming von dem entsprechenden EP benutzt werden würde, also anstatt z.B. "Search" (weil wir ja auch eine Unternehmessuche haben) + die entsprechende HTTP Methode, also eher so in der Richtung: https://github.com/Clarilab/kycnow-v3-docs/blob/main/assets/api-v3/monitoring-manage-changes.png
3. Beim Ordering könnte man eine Loop für das "CheckOrderState" machen, weil der ja bei uns gepollt werden muss

Willi-Pindjukov committed:
update transreg flow png for docu

add new transreg flow png with changed styles and namings

Willi-Pindjukov committed:
change png file

changed some little things in the png file of the diagram of the new transreg flow
